ocl_cuckaroo/src/trimmer.rs

```diff
@@ -47,6 +53,24 @@ macro_rules! kernel_enq(
 		}
 	));
 
+macro_rules! get_device_max_alloc_size {
+	($dev:ident) => {{
+		match $dev.info(DeviceInfo::MaxMemAllocSize) {
```

Comment:
Not sure you need a macro for that (and below), isn't device.info()expect("Failed ...") enough?

Reply:
Makes sense, I'll find a way to work without the macros.